.github/labeler.yml

```diff
@@ -1,40 +1,36 @@
 Documentation:
 - changed-files:
-  - any-glob-to-any-file: docs/**
-
-NuGet:
-- changed-files:
-  - any-glob-to-any-file: '**/*.nuspec'
```

Comment:
This is NuGet related so it should stay in NuGet, not be added to GitHub.

Reply:
It's to reduce the number of label used 
There no need to differentiate Github and Script